Resources/lang/fr/common.php

```diff
@@ -59,56 +59,56 @@
     // Aircraft - Pirep - Flight
     'altitude'  => 'Altitude',
     'avg'       => 'Moyenne',
-    'btime'     => 'Temps de vol (block)',
+    'btime'     => 'Durée de vol (block)',
     'config'    => 'Configuration',
     'dest'      => 'Destination',
     'dest_abr'  => 'Dest',
     'dist'      => 'Distance',
     'flightno'  => 'N° de Vol #',
     'flightid'  => 'Identifiant du vol (Code & Etape)',
-    'ftime'     => 'temps de vol',
-    'fuel'      => 'Carburant',
-    'fuelob'    => 'Carburant à bord',
-    'fuelused'  => 'Carburant utilisé',
+    'ftime'     => 'Heures de vol',
+    'fuel'      => 'Fuel',
+    'fuelob'    => 'Fuel à bord',
+    'fuelused'  => 'Fuel utilisé',
```

Comment:
I would've kept "Carburant" since "fuel" is not french at all and "Carburant" is the literal translation